Description: Allow opening files with Audacity.
 We need to specify that file names are allowed as command line parameter to
 allow opening .aup files with Nautilus and other file managers.
 .
 More details in:
 http://standards.freedesktop.org/desktop-entry-spec/latest/ar01s06.html
Author: Benjamin Drung <bdrung@debian.org>
Bug-Ubuntu: https://launchpad.net/bugs/1006610

--- a/src/audacity.desktop.in
+++ b/src/audacity.desktop.in
@@ -14,7 +14,7 @@
 Type=Application
 Categories=AudioVideo;Audio;AudioVideoEditing;
 
-Exec=@AUDACITY_NAME@
+Exec=@AUDACITY_NAME@ %F
 StartupNotify=false
 Terminal=false
 MimeType=application/ogg;audio/basic;audio/mpeg;audio/x-aiff;audio/x-mp3;audio/x-wav;application/x-audacity-project;
